jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/J2735Node_LL.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright 2018 572682
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License.  You may obtain a copy
+ * of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ ******************************************************************************/
+package us.dot.its.jpo.ode.plugin.j2735;
+
+import java.math.BigDecimal;
+
+import us.dot.its.jpo.ode.plugin.asn1.Asn1Object;
+
+public class J2735Node_LL extends Asn1Object {
+	private static final long serialVersionUID = 1L;
+	
+	private BigDecimal lon;
+	private BigDecimal lat;
+	
+	public J2735Node_LL() {
+		super();
+	}
```

Comment:
Disagree. For POJOs to be used by Jackson, parametertless constructors need to be explicitly defined if constructors with parameters are defined, as one is in this class.  In general, Jackson requires the parameterless constructors, although they might not be explicitly called by other application code, otherwise deserialization may fail.